module/sheets/WitcherActorSheet.js

```diff
@@ -1,8 +1,38 @@
-
 import { RollCustomMessage } from "../chat.js";
+import { witcher } from "../config.js";
 import { getRandomInt, updateDerived, rollSkillCheck, applyWoundTreshold, removeWoundTreshold, genId } from "../witcher.js";
 
 export default class WitcherActorSheet extends ActorSheet {
+    constructor(...args) {
+      super(...args);
+      this.actor.update({
+        "data.CritDescription": witcher.CritDescription,
+      });
+      this.actor.update({
+        "data.CritModDescription": witcher.CritModDescription,
+      });
+      this.actor.update({
+        "data.CritGravity": witcher.CritGravity,
+      });
+      this.actor.update({
+        "data.CritGravityDefaultEffect": witcher.CritGravityDefaultEffect,
+      });
+      this.actor.update({
+        "data.CritMod": witcher.CritMod,
+      });
+      this.actor.update({
+        "data.CritSimple": witcher.CritSimple,
+      });
+      this.actor.update({
+        "data.CritComplex": witcher.CritComplex,
+      });
+      this.actor.update({
+        "data.CritDifficult": witcher.CritDifficult,
+      });
+      this.actor.update({
+        "data.CritDeadly": witcher.CritDeadly,
+      });
```

Comment:
This should be part of a migration instead of always calling 7 updates whenever a character sheet is opened.

Reply:
Hey there, thanks for looking through, how do you mean by migration?

Reply:
There's no point updating the actor with static information more than once.
Migration updates the data once when it's needed (e.g. https://gitlab.com/foundrynet/dnd5e/-/blob/master/module/migration.js)

Reply:
Thanks for explaining. I did eventually just remove these 7 updates as they were actually redundant. Though I found that these 7 updates are only fired once per character sheet for every browser session when they are open for the first time.

As for migration.js, I don't think there is a module for that here. And I have no clue how to go about implementing it that way for this project yet.

Reply:
For your information you could have done one update:
this.actor.update({
        "data.CritDescription": witcher.CritDescription,
        "data.CritModDescription": witcher.CritModDescription,
        "data.CritGravity": witcher.CritGravity,
        ...
      });